README.md

```diff
@@ -26,10 +26,12 @@ to join/post/read messages on the
 
 CI System      | Build System | Platform | Status
 -------------- | ------------ | -------- | ------
-GitHub Actions | CMake        | Linux    | [![github-cmake-linux-status](https://github.com/google/iree/workflows/CMake%20Build/badge.svg)](https://github.com/google/iree/actions?query=branch%3Amaster+event%3Apush+workflow%3A%22CMake+Build%22)
-GitHub Actions | Bazel        | Linux    | [![github-bazel-linux-status](https://github.com/google/iree/workflows/Bazel%20Build/badge.svg)](https://github.com/google/iree/actions?query=branch%3Amaster+event%3Apush+workflow%3A%22Bazel+Build%22)
+GitHub Actions | CMake        | Linux    | [Workflow History](https://github.com/google/iree/actions?query=event%3Apush+workflow%3A%22CMake+Build%22)*
+GitHub Actions | Bazel        | Linux    | [Workflow History](https://github.com/google/iree/actions?query=event%3Apush+workflow%3A%22Bazel+Build%22)*
 Kokoro         | Bazel        | Linux    | [![kokoro-status-bazel-linux](https://storage.googleapis.com/iree-oss-build-badges/bazel/build_status_linux.svg)](https://storage.googleapis.com/iree-oss-build-badges/bazel/build_result_linux.html)
 
+\* Due to a [bug in GitHub actions](https://github.community/t5/GitHub-Actions/Actions-Reporting-Wrong-Branch-on-Merges/td-p/36966) runs will frequently be tagged with the branch that was pushed from instead of the master branch. This also breaks the status badge.
```

Comment:
This is sort of noisy to include in the README. Want to omit it?

Reply:
Sure. I want to put it *somewhere* though, since GitHub actions are documented as having a status badge and folks might reasonably ask why we're not using that...

Lemme see if I can come up with something less noisy

Reply:
I just deleted it. GitHub markdown doesn't appear to support footnotes or tooltips